vector/src/main/java/im/vector/app/features/settings/devices/v2/overview/SessionOverviewViewEvent.kt

```diff
@@ -20,6 +20,7 @@ import im.vector.app.core.platform.VectorViewEvents
 import org.matrix.android.sdk.api.session.Session
 
 sealed class SessionOverviewViewEvent : VectorViewEvents {
-    data class SelfVerification(val session: Session) : SessionOverviewViewEvent()
+    data class ShowVerifyCurrentSession(val session: Session) : SessionOverviewViewEvent()
```

Comment:
Just a small remark: please remove the session from here.

Reply:
Yes thanks, I removed the argument.